Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -159,7 +159,7 @@ protected function registerAllTagsToFlushForNodeInWorkspace(NodeInterface $node,
     {
         // Ensure that we're dealing with the variant of the given node that actually
         // lives in the given workspace
-        if ($node->getWorkspace()->getName() !== $workspace->getName()) {
+        if ($node->isRemoved() === false && $node->getWorkspace()->getName() !== $workspace->getName()) {
             $workspaceContext = $this->contextFactory->create(
                 array_merge(
                     $node->getContext()->getProperties(),
```

Comment:
or should we use showRemovedNodes in the context or something?

fyi will also run the neos e2e tests against this soon

Reply:
This is not working, I also tried that.